packages/repo-tools/src/commands/peer-deps/peer-deps.ts

```diff
@@ -104,6 +104,11 @@ export default async ({ fix }: { fix: boolean }) => {
   const packagesWithRelevantDependencies = [];
 
   for (const pkg of packages) {
+    if (pkg.dir.includes('canon-website')) {
```

Comment:
Rather than special-casing the canon-website specifically, should we perhaps filter out packages that don't have a Backstage role? We could rename `isStandaloneApplication` to `isBackstagePackage`, invert the logic, and add the clause in there.

Reply:
We removed this change to avoid adding the new website as a workspace.